core/src/main/java/org/apache/struts2/components/Token.java

```diff
@@ -66,7 +66,7 @@ protected String getDefaultTemplate() {
     /**
      * First looks for the token in the PageContext using the supplied name (or {@link org.apache.struts2.util.TokenHelper#DEFAULT_TOKEN_NAME}
      * if no name is provided) so that the same token can be re-used for the scope of a request for the same name. If
-     * the token is not in the PageContext, a new Token is created and set into the Session and the PageContext with
+     * the token is not in thre PageContext, a new Token is created and set into the Session and the PageContext with
```

Comment:
I think this might be a typo 
```suggestion
     * the token is not in the PageContext, a new Token is created and set into the Session and the PageContext with
```